src/Microsoft.Framework.Runtime.Hosting/Internal/DefaultPackagePathResolver.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using NuGet.Versioning;
+
+namespace Microsoft.Framework.Runtime.Internal 
+{
+    public class DefaultPackagePathResolver
```

Comment:
It might make sense to just have a single method on here to force the fact that the runtime has no dependencies on NuGet per set.